LikelihoodToRespond.md

```diff
@@ -0,0 +1,70 @@
+Likelihood To Respond Service
+=============
+
+Table of Contents
+_________
+- [Overview](#overview)
+- [Request Information](#request-information)
+- [Response Information](#response-information)
+- [Sample Request](#sample-request)
+
+# Overview
+
+The Likelihood To Respond (LTR) service provides data to describe the likelihood that a job seeker will respond positively when contacted regarding a new job opportunity. Data returned by the service is computed ahead of time by the Search Data Science team and is refreshed nightly.
```

Comment:
yeah, I was confused about when to use "to", now I got it, when regard plays as a verb, no to needed.